specs/~inheritance.yml

```diff
@@ -0,0 +1,225 @@
+overview: |
+  Like partials, Parent tags are used to expand an external template into the
+  current template. Contrary to partials, Parent tags may contain optional
+  arguments delimited by block tags. For this reason, Parent tags may also be
+  referred to as Parametric Partials.
+
+  The Parent tags' content MUST be a non-whitespace character sequence NOT
+  containing the current closing delimiter; each Parent tag MUST be followed by
+  an End Section tag with the same content within the matching Parent tag.
+
+  This tag's content names the Parent template to inject. Set Delimiter tags
+  MUST NOT affect the parsing of a Parent template. The Parent MUST be rendered
+  against the context stack local to the tag. If the named Parent cannot be
+  found, the empty string SHOULD be used instead, as in interpolations.
+
+  Parent tags SHOULD be treated as standalone when appropriate. If this tag is
+  used standalone, any whitespace preceding the tag should be treated as
+  indentation, and prepended to each line of the Parent before rendering.
+
+  The Block tags' content MUST be a non-whitespace character sequence NOT
+  containing the current closing delimiter. Each Block tag MUST be followed by
+  an End Section tag with the same content within the matching Block tag. This
+  tag's content determines the parameter or argument name.
+
+  Block tags may appear both inside and outside of Parent tags. In both cases,
+  they specify a position within the template that can be overridden; it is a
+  parameter of the containing template. The template text between the Block tag
+  and its matching End Section tag defines the default content to render when
+  the parameter is not overridden from outside.
+
+  In addition, when used inside of a Parent tag, the template text between a
+  Block tag and its matching End Section tag defines content that replaces the
+  default defined in the Parent template. This content is the argument passed
+  to the Parent template.
+
+  The practice of injecting an external template using a Parent tag is referred
+  to as inheritance. If the Parent tag includes a Block tag that overrides a
+  parameter of the Parent template, this may also be referred to as
+  substitution.
```

Comment:
I'm not sure this paragraph needs to exist to make the spec work.

Reply:
I feel that it clarifies why the file is called "inheritance" at all, and why the word "substitution" is used in some of the specs. After my discussion with @gasche, I feel that `~parametric-partial.yml` would actually be more straightforward for new people to understand where the name is coming from. If we remove this paragraph, I feel that the word "substitution" should also be removed from all specs.